README.md

```diff
@@ -0,0 +1,73 @@
+# Environment Variables Extension 
+Server extension to get array environment variables or get environment variable value by key.
+
+## Getting started
+
+Install [nvm](https://github.com/creationix/nvm#install-script).
+
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.5/install.sh | bash
+
+Install npm and node.
+
+    nvm install 8
+    nvm use 8
+
+Install yarn.
+
+    npm install -g yarn
+
+## Running in the browser
+
+    yarn rebuild:browser
+    cd browser-app
+    yarn start
+
+Open http://localhost:3000 in the browser.
+
+## Running in the Electron
+
+    yarn rebuild:electron
+    cd electron-app
+    yarn start
+
+## Developing with the browser example
+
+Start watching of the environment variables extension.
+
+    cd env-variables-extension
+    yarn watch
+
+Start watching of the browser example.
+
+    yarn rebuild:browser
+    cd browser-app
+    yarn watch
+
+Launch `Start Browser Backend` configuration from VS code.
+
+Open http://localhost:3000 in the browser.
+
+## Developing with the Electron example
+
+Start watching of the hello world extension.
```

Comment:
It's not a `hello world extension` )
FYI: there was a bug in the Yeoman generator for Theia extensions which I've already [fixed](https://github.com/theia-ide/generator-theia-extension/pull/15) a couple of weeks ago.